src/openrct2/localisation/string_ids.h

```diff
@@ -3651,6 +3651,9 @@ enum {
 	STR_SET_MONEY = 5999,
 	STR_ENTER_NEW_VALUE = 6000,
 
+	STR_QUICK_DEMOLISH_RIDE = 6001,
+	STR_PLEASE_SLOW_DOWN = 6002,
```

Comment:
@marijnvdwerf 
The idea was to keep it intentionally generic, that way it can be used in multiple cases within the network (for example, it is now also used with the cluster scenery detection, rather than saying "Can't do this: Can't do this")